docs/requirements.txt

```diff
@@ -0,0 +1,7 @@
+sphinx==7.1.2
+sphinx-rtd-theme==1.3.0rc1
```

Comment:
Why did you pick the rc?

Reply:
but you don't need the rc to get it to work

Reply:
I wasn't taking any chances!
But OK, removed the rc now.